library/IvozProvider/Mapper/Sql/Users.php

```diff
@@ -20,45 +20,52 @@
 namespace IvozProvider\Mapper\Sql;
 class Users extends Raw\Users
 {
-    protected function _save(\IvozProvider\Model\Raw\Users $model,
-        $recursive = false, $useTransaction = true, $transactionTag = null, $forceInsert = false
+    protected function _save(
+        \IvozProvider\Model\Raw\Users $model,
+        $recursive = false,
+        $useTransaction = true,
+        $transactionTag = null,
+        $forceInsert = false
     )
     {
         $isNew = !$model->getPrimaryKey();
 
-        // '' is NULL (avoid triggering the UNIQUE KEY)
-        if ($model->getEmail() == '') $model->setEmail(null);
+        if ($model->getEmail() === '') {
+            // '' is NULL (avoid triggering the UNIQUE KEY)
+            $model->setEmail(null);
+        }
 
         if ($isNew) {
             // Sane defaults for hidden fields
-            if (!$model->hasChange('timezoneId'))
-                $model->setTimezoneId($model->getCompany()->getBrand()->getDefaultTimezoneId());
-
-            if (!$model->hasChange('exceptionBoosAssistantRegExp'))
-                $model->setExceptionBoosAssistantRegExp('');
+            $model->setTimezoneId(
```

Comment:
All the _hasChange fiesta_ is to avoid overriding values in API created items.

I'm afraid they are still necessary, aren't they?